superset/assets/src/SqlLab/components/SqlEditorLeftBar.jsx

```diff
@@ -83,17 +83,9 @@ export default class SqlEditorLeftBar extends React.PureComponent {
       this.setState({ tableName: '' });
       return;
     }
-    const namePieces = tableOpt.value.split('.');
-    let tableName = namePieces[0];
-    let schemaName = this.props.queryEditor.schema;
-    if (namePieces.length === 1) {
-      this.setState({ tableName });
-    } else {
-      schemaName = namePieces[0];
-      tableName = namePieces[1];
-      this.setState({ tableName });
-      this.props.actions.queryEditorSetSchema(this.props.queryEditor, schemaName);
-    }
+    const tableName = tableOpt.value;
+    const schemaName = this.props.queryEditor.schema;
+    this.setState({ tableName });
```

Comment:
A small comment here @mistercrunch : It's my understanding that some databases pass on the schema name along with the table name separated by a period (I believe BigQuery does, and Presto sometimes, too). It was apparently assumed that a period *always* indicates `schema.table`. However, periods are supported in many dbs schema/table names, hence they can't be universally removed. I don't removing this will have adverse side-effects, other than showing the shema name along with the table name on some dbs.